src/snowflake/cli/api/project/definition.py

```diff
@@ -102,16 +104,16 @@ def generate_local_override_yml(
     return project.update_from_dict(local)
 
 
-def default_app_package(project_name: str):
+def resource_suffix() -> str:
+    """A suffix that should be added to account-level resources."""
+    return os.environ.get(SNOWFLAKE_CLI_RESOURCE_SUFFIX_VAR, "")
```

Comment:
Should this really be part of the public API? At least we should give it a very descriptive name, I normally worry about making non-prod logic sound so prod-like.

Reply:
based on our discussion earlier, it seemed to me that this would indeed be part of the public API (but undocumented). Let's PL this for tomorrow morning